chemprop/cli/utils/parsing.py

```diff
@@ -197,28 +198,78 @@ def make_datapoints(
     else:
         N = len(smiss[0])
 
+    if len(smiss) > 0:
+        molss = [[make_mol(smi, keep_h, add_h) for smi in smis] for smis in smiss]
+    if len(rxnss) > 0:
+        rctss = [
+            [
+                make_mol(f"{rct_smi}.{agt_smi}" if agt_smi else rct_smi, keep_h, add_h)
```

Comment:
Is the reagent considered in the reactant complex documented somewhere else?

Reply:
I make a note of it [here](https://chemprop.readthedocs.io/en/main/tutorial/python/data/datapoints.html#Reaction-Datapoints)

Could also be good to put in the docstring for `_ReactionDatapointMixin` but they also don't show up in the docs yet because we still need to enable autoapi for private classes.